### PR TITLE
Issue/621/small app logic changes

### PR DESCRIPTION
**Pull request by Bedpaw**
- [x] Hide guide => featureToggle flag === OLD_GUIDE
- [x] Fix jumping icon when enter event
- [x] Remove empty column pointKey for common users
- [x] Correct translation in user list
- [x] Add feature toggle for user popup === USER_POPUP
- [x] Add nickname to event store
- [x] Correct menu header
- [x] Remove points category split by pointType
- [x] Change observer route from collectedPoints to scoreboard
- [x] Fix eventDuration mapping in checkout session and checkEvent

### CODE REVIEW

File: client/src/components/organisms/header.vue [hunk 1]
```diff
@@ -30,10 +30,11 @@
       class="m-panel f-header f-side f-right"
     >
       <a-icon
-        v-if="$route.name === ROUTES.eventsList.name"
+        v-if="$route.name === ROUTES.eventsList.name && !eventId"
```
Comment (dbetka) [inline client/src/components/organisms/header.vue:33]: `!eventId` is not easy to read. Should be checked if it's not undefined or null (related to possible types).
Different solution is to add getter to check if event is not defined.

File: client/src/components/organisms/popup/event-confirmation.vue [hunk 2]
```diff
@@ -75,6 +75,7 @@ export default {
         eventEndDate,
         role,
       } = this.event;
+      console.log(this.event);
```
Comment (dbetka) [inline client/src/components/organisms/popup/event-confirmation.vue:78]: Remove console log

**Review by dbetka — CHANGES_REQUESTED** [review]
If I try to get `http://localhost:3030/users-list?USER_POPUP=true` router push me to `http://localhost:3030/events-list` so I'm not able to test featureToggle flags.

Build errors and warnings:
image

File: client/src/components/organisms/header.vue (hunk 1 above)
Comment (Bedpaw) [reply client/src/components/organisms/header.vue]: done, changed to === null

File: client/src/components/organisms/popup/event-confirmation.vue (hunk 2 above)
Comment (Bedpaw) [reply client/src/components/organisms/popup/event-confirmation.vue]: removed

**Review by dbetka — APPROVED** [review]
feature toggles are checked.

Rest look OK :)

**Pull request merged**